apps/web-giddh/src/app/shared/header/header.component.html

```diff
@@ -263,7 +263,7 @@ <h6>Downloads</h6>
     <div class="menu-wrapper">
         <!-- switch branch start -->
         <div class="switch-branch-wrapper">
-            <div class="btn-group" dropdown placement="bottom right" [autoClose]="false">
+            <div class="btn-group" dropdown placement="bottom right" [autoClose]="true">
```

Comment:
This will not allow the user to search the branches and will close the menu, please check

Reply:
Fixed